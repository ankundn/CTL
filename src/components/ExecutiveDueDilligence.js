import React from "react";
import {
  
  MDBAnimation
} from "mdbreact";



const ExecutiveDueDilligence = () => {
  return (
   

<div className="container z-depth-1 my-5 py-5">

<h6 className="font-weight-bold text-center grey-text text-uppercase small mb-4">Services</h6>
<MDBAnimation className="clientanimation"type="bounceIn" duration="3s">
  <h2 className="font-weight-bold text-center dark-grey-text pb-2">Executive Due Dilligence Services</h2>
            </MDBAnimation>
<hr className="w-header my-4" />
<p className="lead text-center text-muted pt-2 mb-5"> We conduct due
         diligence, on behalf of clients, on executive level staff or
         candidates, as well as on key persons at prospective investee
         companies or partners.</p>
         <p className="lead text-center text-muted pt-2 mb-5">Our executive due diligence investigations may
     be tailored to cover the following areas:</p>
<section>
<div className="row d-flex justify-content-center">

<div className="col-md-6 col-lg-5 col-xl-4">
      <h5 className="font-weight-normal border-top border-primary pt-4 mb-4">Employment history</h5>
      <p className="text-muted mb-5 pb-md-3">Verification and confirmation that the employement information on a job application is accurate.</p>
    </div>

    <div className="col-md-6 col-lg-5 col-xl-4">
      <h5 className="font-weight-normal border-top border-primary pt-4 mb-4">Academic and professional license qualifications</h5>
      <p className="text-muted mb-5 pb-md-3">Verification and confirmation that academic and professional license qualifications are legitimate.</p>
    </div>

  <div className="w-100"></div>

  <div className="col-md-6 col-lg-5 col-xl-4">
      <h5 className="font-weight-normal border-top border-primary pt-4 mb-4">History of criminal and civil litigation</h5>
      <p className="text-muted mb-5 pb-md-3">Background check on a person to see if they have any history of trial in criminal court or if they have had any civil suit brought agianst them.</p>
    </div>
    

</div>



<div className="row d-flex justify-content-center">

<div className="col-md-6 col-lg-5 col-xl-4">
      <h5 className="font-weight-normal border-top border-primary pt-4 mb-4">Reputation Risks</h5>
      <p className="text-muted mb-5 pb-md-3">Establish threats or danger to the good name or standing of a business or entity</p>
    </div>

    <div className="col-md-6 col-lg-5 col-xl-4">
      <h5 className="font-weight-normal border-top border-primary pt-4 mb-4">Client/Supplier Relationships</h5>
      <p className="text-muted mb-5 pb-md-3">Establish and explain the relationship between a client and the supplier</p>
    </div> 

<div className="w-100"></div>


    
<div className="col-md-6 col-lg-5 col-xl-4">
      <h5 className="font-weight-normal border-top border-primary pt-4 mb-4">Source of Wealth</h5>
      <p className="text-muted mb-5 pb-md-3">Investigation on how one got their funds.</p>
    </div>

</div>




<div className="row d-flex justify-content-center">


  

<div className="w-100"></div>

<div className="col-md-6 col-lg-5 col-xl-4">
      <h5 className="font-weight-normal border-top border-primary pt-4 mb-4">Conflict of Interest</h5>
      <p className="text-muted mb-5 pb-md-3">Establish whether an entity or individual is unreliable because of a clash
      between personal interests and professional duties or responsibilities.
      </p>
    </div>  

    <div className="col-md-6 col-lg-5 col-xl-4">
      <h5 className="font-weight-normal border-top border-primary pt-4 mb-4">Regulatory Violations</h5>
      <p className="text-muted mb-5 pb-md-3">Violation of the Securities Act of 1933 or the Securities Exchange Act of 1934 by an Insured, in his or her capacity as such, related to financial statement reporting and disclosure to investors including violations of Generally Accepted Accounting Principles (“GAAP”).</p>
    </div> 
   
</div>



{/* <div className="row d-flex justify-content-center">

<div className="col-md-6 col-lg-5 col-xl-4">
      <h5 className="font-weight-normal border-top border-primary pt-4 mb-4">Links to Organized Crime</h5>
      <p className="text-muted mb-5 pb-md-3">Explanation on Links to Organized Crime</p>
    </div> 
    <div className="col-md-6 col-lg-5 col-xl-4">
      <h5 className="font-weight-normal border-top border-primary pt-4 mb-4">Verification of Credentials</h5>
      <p className="text-muted mb-5 pb-md-3">Explanation on Verification of Credentials</p>
    </div> 

<div className="w-100"></div>
<div className="col-md-6 col-lg-5 col-xl-4">
      <h5 className="font-weight-normal border-top border-primary pt-4 mb-4">Debts and Solvency</h5>
      <p className="text-muted mb-5 pb-md-3">Explanation on Debts and Solvency</p>
    </div> 
    <div className="col-md-6 col-lg-5 col-xl-4">
      <h5 className="font-weight-normal border-top border-primary pt-4 mb-4">Judgements</h5>
      <p className="text-muted mb-5 pb-md-3">Explanation on Judgements</p>
    </div> 

</div> */}

<div className="row d-flex justify-content-center">

<div className="col-md-6 col-lg-5 col-xl-4">
      <h5 className="font-weight-normal border-top border-primary pt-4 mb-4">And So Many More Executive Due Diligence Services</h5>
      <p className="text-muted mb-5 pb-md-3">At Corporate Technical Services we have more executive due diligence services offered that include; 
      Extensive Reference Interviews, Interpol Search, Links To Organized Crime, Verification of Credentials, Debts and Solvency, Judgements etc
      </p>
    </div> 

<div className="w-100"></div>


</div>
</section>


</div>

   
    




   
  );
};

export default ExecutiveDueDilligence;
