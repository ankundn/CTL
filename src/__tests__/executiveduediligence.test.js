import React from "react";
import renderer from "react-test-renderer";
import ExecutiveDueDilligence from "../components/ExecutiveDueDilligence";

test(" Executive Due Diligence renders correctly", () => {
  const tree = renderer.create(<ExecutiveDueDilligence />).toJSON();
  expect(tree).toMatchSnapshot();
});