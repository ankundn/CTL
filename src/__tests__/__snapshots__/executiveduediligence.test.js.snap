// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Executive Due Diligence renders correctly 1`] = `
<div
  className="container z-depth-1 my-5 py-5"
>
  <h6
    className="font-weight-bold text-center grey-text text-uppercase small mb-4"
  >
    Services
  </h6>
  <div
    className="animated bounceIn clientanimation"
    data-test="animation"
    onAnimationEnd={[Function]}
    onAnimationIteration={[Function]}
    onAnimationStart={[Function]}
    style={
      Object {
        "animationDelay": undefined,
        "animationDuration": "3s",
        "animationIterationCount": 1,
        "animationName": "bounceIn",
        "visibility": "visible",
      }
    }
  >
    <h2
      className="font-weight-bold text-center dark-grey-text pb-2"
    >
      Executive Due Dilligence Services
    </h2>
  </div>
  <hr
    className="w-header my-4"
  />
  <p
    className="lead text-center text-muted pt-2 mb-5"
  >
     We conduct due diligence, on behalf of clients, on executive level staff or candidates, as well as on key persons at prospective investee companies or partners.
  </p>
  <p
    className="lead text-center text-muted pt-2 mb-5"
  >
    Our executive due diligence investigations may be tailored to cover the following areas:
  </p>
  <section>
    <div
      className="row d-flex justify-content-center"
    >
      <div
        className="col-md-6 col-lg-5 col-xl-4"
      >
        <h5
          className="font-weight-normal border-top border-primary pt-4 mb-4"
        >
          Employment history
        </h5>
        <p
          className="text-muted mb-5 pb-md-3"
        >
          Verification and confirmation that the employement information on a job application is accurate.
        </p>
      </div>
      <div
        className="col-md-6 col-lg-5 col-xl-4"
      >
        <h5
          className="font-weight-normal border-top border-primary pt-4 mb-4"
        >
          Academic and professional license qualifications
        </h5>
        <p
          className="text-muted mb-5 pb-md-3"
        >
          Verification and confirmation that academic and professional license qualifications are legitimate.
        </p>
      </div>
      <div
        className="w-100"
      />
      <div
        className="col-md-6 col-lg-5 col-xl-4"
      >
        <h5
          className="font-weight-normal border-top border-primary pt-4 mb-4"
        >
          History of criminal and civil litigation
        </h5>
        <p
          className="text-muted mb-5 pb-md-3"
        >
          Background check on a person to see if they have any history of trial in criminal court or if they have had any civil suit brought agianst them.
        </p>
      </div>
    </div>
    <div
      className="row d-flex justify-content-center"
    >
      <div
        className="col-md-6 col-lg-5 col-xl-4"
      >
        <h5
          className="font-weight-normal border-top border-primary pt-4 mb-4"
        >
          Reputation Risks
        </h5>
        <p
          className="text-muted mb-5 pb-md-3"
        >
          Establish threats or danger to the good name or standing of a business or entity
        </p>
      </div>
      <div
        className="col-md-6 col-lg-5 col-xl-4"
      >
        <h5
          className="font-weight-normal border-top border-primary pt-4 mb-4"
        >
          Client/Supplier Relationships
        </h5>
        <p
          className="text-muted mb-5 pb-md-3"
        >
          Establish and explain the relationship between a client and the supplier
        </p>
      </div>
      <div
        className="w-100"
      />
      <div
        className="col-md-6 col-lg-5 col-xl-4"
      >
        <h5
          className="font-weight-normal border-top border-primary pt-4 mb-4"
        >
          Source of Wealth
        </h5>
        <p
          className="text-muted mb-5 pb-md-3"
        >
          Investigation on how one got their funds.
        </p>
      </div>
    </div>
    <div
      className="row d-flex justify-content-center"
    >
      <div
        className="w-100"
      />
      <div
        className="col-md-6 col-lg-5 col-xl-4"
      >
        <h5
          className="font-weight-normal border-top border-primary pt-4 mb-4"
        >
          Conflict of Interest
        </h5>
        <p
          className="text-muted mb-5 pb-md-3"
        >
          Establish whether an entity or individual is unreliable because of a clash between personal interests and professional duties or responsibilities.
        </p>
      </div>
      <div
        className="col-md-6 col-lg-5 col-xl-4"
      >
        <h5
          className="font-weight-normal border-top border-primary pt-4 mb-4"
        >
          Regulatory Violations
        </h5>
        <p
          className="text-muted mb-5 pb-md-3"
        >
          Violation of the Securities Act of 1933 or the Securities Exchange Act of 1934 by an Insured, in his or her capacity as such, related to financial statement reporting and disclosure to investors including violations of Generally Accepted Accounting Principles (“GAAP”).
        </p>
      </div>
    </div>
    <div
      className="row d-flex justify-content-center"
    >
      <div
        className="col-md-6 col-lg-5 col-xl-4"
      >
        <h5
          className="font-weight-normal border-top border-primary pt-4 mb-4"
        >
          And So Many More Executive Due Diligence Services
        </h5>
        <p
          className="text-muted mb-5 pb-md-3"
        >
          At Corporate Technical Services we have more executive due diligence services offered that include; Extensive Reference Interviews, Interpol Search, Links To Organized Crime, Verification of Credentials, Debts and Solvency, Judgements etc
        </p>
      </div>
      <div
        className="w-100"
      />
    </div>
  </section>
</div>
`;
